src/api/eventTypes.js

```diff
@@ -144,3 +145,9 @@ export type UpdateMessageFlagsEvent = {|
   all: boolean,
   messages: number[],
 |};
+
+export type MutedUsersEvent = {|
```

Comment:
nit: we've had the event types in alphabetical order in this file (same as in the `EventTypes` enum/class), so let's maintain that

Reply:
This file isn't quite in alphabetical order (since `Submessage` wants to be next to `Message`, it seems?), but I made it a bit closer :)

Reply:
Hmm true! And `UserStatusEvent` in what seems like a completely arbitrary place. Well, thanks for making it closer :slightly_smiling_face: